### PR TITLE
export contractsinfo to deployment

### PR DESCRIPTION
**Pull request by xgaozoyoe**

### CODE REVIEW

**Comment by yymone** [comment]
should remove those js and js.map files generate from ts file?

File: clients/config-contracts-info.ts [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        name:"tToken"
+      },
+    ]
+}
+
+const fs = require("fs");
+const path = require("path");
+
+fs.writeFileSync(
+    path.resolve(__dirname, "../../deployment/config", "contracts-info.json"),
```
Comment (yymone) [inline clients/config-contracts-info.ts:66]: should use node_module's deployment I think?

File: clients/contracts/bridge.ts [hunk 2]
```diff
@@ -3,11 +3,9 @@ import { DelphinusContract, DelphinusWeb3 } from "web3subscriber/src/client";
 import { decodeL1address } from "web3subscriber/src/addresses";
 import { PromiseBinder } from "web3subscriber/src/pbinder";
 import { TokenContract } from "./token";
-import { Tokens, Chains } from "./tokenlist";
-import { extraTokens} from "delphinus-deployment/config/extratokens";
-const BridgeContractABI = require("../../build/contracts/Bridge.json");
+import { extraTokens, Chains, contractsInfo } from "delphinus-deployment/config/contractsinfo";
```
Comment (yymone) [inline clients/contracts/bridge.ts:6]: First, in config-contracts-info.ts, the file name is contracts-info.json.

Comment (xgaozoyoe) [reply clients/contracts/bridge.ts:6]: it was in https://github.com/DelphinusLab/delphinus-deployment/commit/c6311ce4d33b84b2efe2fc8df5e4f31dbbd2be81

**Comment by yymone** [comment]
Looks good for me. Just need clean js and js.map

**Pull request merged**